<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowRank</class>
 <widget class="QDialog" name="WindowRank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButtonTwo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Оберіть ранг карти</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonThree">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonFour">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonFive">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSix">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSeven">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonEight">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonNine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonTen">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonJack">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Валєт</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonQueen">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дама</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonKing">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Король</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonAce">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Туз</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
